lib/hooks/useSearchRepos.ts

```diff
@@ -1,7 +1,68 @@
 import useSWR, { Fetcher } from "swr";
-import githubApiFetcher from "lib/utils/github-api-fetcher";
+import { supabase } from "lib/utils/supabase";
+
+const baseUrl = "https://api.github.com";
```

Comment:
Should this use the env found in `NEXT_PUBLIC_API_URL`?

Reply:
Not on this one because this is the base for the GitHub API and not ours